.github/workflows/build.yml

```diff
@@ -205,6 +205,42 @@ jobs:
         run: |
           cat ./build/meson-logs/testlog.txt
 
+  linux-ffmpeg-4-4:
+    runs-on: ubuntu-22.04 # pin this to 22.04 to test 4.4 specifically
```

Comment:
this works until github removes the image
we can choose not to care because that will be a while and we probably have bumped the requirement by that time (or maybe not?)
**or** you could move this into the [matching docker image](https://hub.docker.com/_/ubuntu)

Reply:
They still carry 20.04, so I would say they'd keep this around for another 2 years. By that point surely the minimum would be bumped...